homeassistant/components/lidarr/sensor.py

```diff
@@ -100,6 +100,14 @@ class LidarrSensorEntityDescription(
             album.title: album.artist.artistName for album in data.records
         },
     ),
+    "albums": LidarrSensorEntityDescription[int](
+        key="albums",
+        translation_key="albums",
+        native_unit_of_measurement="Albums",
```

Comment:
```suggestion
        native_unit_of_measurement="albums",
```
We want lowercase UoMs, but I do see that the rest doesn't have this, so that would be a nice followup